Fix-ups and remote host support for tests/install_vm.py

comps opened the pull request:
See the commit messages for each separate commit for more details.

The `install_vm.py` tool lacked support for remote libvirt hosts and also had a bunch of questionable options and hardcoded defaults - I've tried to clean it up to the best of my knowledge and it should now work for both local and remote installs.

If you use this tool in some non-default / non-standard way, please test these changes and let me know if something breaks!

pep8speaks commented:
Hello @comps! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`tests/install_vm.py`](https://github.com/ComplianceAsCode/content/blob/3e5430fb4837875cc9f8475abf8637dc9389c319/tests/install_vm.py):

> [Line 121:100](https://github.com/ComplianceAsCode/content/blob/3e5430fb4837875cc9f8475abf8637dc9389c319/tests/install_vm.py#L121): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (366 > 99 characters)
> [Line 133:100](https://github.com/ComplianceAsCode/content/blob/3e5430fb4837875cc9f8475abf8637dc9389c319/tests/install_vm.py#L133): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 99 characters)
> [Line 135:100](https://github.com/ComplianceAsCode/content/blob/3e5430fb4837875cc9f8475abf8637dc9389c319/tests/install_vm.py#L135): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (139 > 99 characters)
> [Line 137:100](https://github.com/ComplianceAsCode/content/blob/3e5430fb4837875cc9f8475abf8637dc9389c319/tests/install_vm.py#L137): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (233 > 99 characters)

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/ee05c8e6-2691-4f70-9e49-512e96dc7a11) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/ee05c8e6-2691-4f70-9e49-512e96dc7a11) completed: **1 new issues, 1 updated code elements**

dahaic commented:
@openscap-ci add to whitelist

dahaic commented:
@openscap-ci test this please

jan-cerny commented on tests/install_vm.py:
```diff
@@ -19,13 +19,6 @@ def parse_args():
         help="Path to a kickstart file for installation of a VM."
     )
-    parser.add_argument(
-        "--distro",
```
I prefer keeping this option, because it's an user friendly feature, the purpose of it is that users don't have to search for URLs of Fedora or CentOS the internet. That's convenient for them. You can use `--url` anytime to override the default hard coded URL if you want.

I suggest removing only `choices` keyword to not limit the choices of distributions and removing the `data.variant` variable, but keeping the `--distro`. Possibly, you can extend or update the hard coded URls.

comps replied:
Ok, my problem with it was that it was directly related to `--variant` for `virt-install`; having it solely as an URL convenience (without the `virt-install` snippet) makes sense. Thanks!

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/eb90aa0e-1f2a-4ab4-a6c7-783438dc846b) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/ComplianceAsCode/content/inspections/eb90aa0e-1f2a-4ab4-a6c7-783438dc846b) completed: **2 new issues**

comps commented:
Rebased, changes since last push:
- f6a8fb1: added format=qcow2 to default disk_spec; while installation would work fine, the current test suite needs internal snapshots, so it would just fail later on
- 1db3559: added back --distro as an URL alias only (fedora/centos7)
- e89cb5b: added --disk for more advanced storage specification (pools)

jan-cerny commented:
@comps Thank you for your patches!
